app/views/blog_posts/show.html.erb

```diff
@@ -6,9 +6,9 @@
         data: { turbo_method: :delete },
         class: "btn btn-error btn-sm"
       %>
-    <% elsif policy(@blog_post.build_featured_blog_post).create? %>
+    <% elsif policy(@blog_post.initialize_feature).create? %>
```

Comment:
I am pretty sure we had a similar issue somewhere in the past with the `policy` view helper not being allowed to route to a specific policy (it has to be what the record inference is). Is this the reason you do this round about way?

If so, I think at this point we might as well add our own helper that somewhat overrides this to offer the correct functionality.

Reply:
I will also say this is one of the few use cases that we can have a non crud policy check because this is super specific to what it does.

Reply:
Yeah this was purely with the intent to keep the policies CRUD. If you're good with a `#feature?` policy method on blog posts, I could absolutely do it that way.